### DISCUSSION TITLE
How to add header cell filter? Is it possible?

### DISCUSSION THREAD

**Discussion by ivan-po-mars** [nick-keller/react-datasheet-grid#217] [Q&A] [answered]
![Screen Shot 2022-11-30 at 12 36 20](https://user-images.githubusercontent.com/55884410/204785529-52f115f3-fa99-4029-a4e0-c8d6340975ed.png)

**Comment by nick-keller** [comment, marked as answer]
You can use the [title](https://react-datasheet-grid.netlify.app/docs/api-reference/columns#title) of the column to pass your custom component

**Reply by ivan-po-mars** [reply]
I'm trying but have issues with input focus and change handler. Always select and change the first cell
![Screen Shot 2022-11-30 at 13 45 29](https://user-images.githubusercontent.com/55884410/204788479-1ada9910-c1e8-49f5-8255-38dff91ee055.png)
